Astro CLI 1.2 docs (#855)
astronomer/docs: software/cli-release-notes.md

> If you have any questions or a bug to report, reach out to us via [Astronomer Support](https://support.astronomer.io).

Reviewer: ```suggestion
If you have any questions or a bug to report, contact [Astronomer Support](https://support.astronomer.io).
```

## Astro CLI v1.2.0

Release date: June 27, 2022

### A Shared CLI for All Astronomer Users

:::danger Breaking Change

> Astro CLI v1.2.0 includes breaking changes that might effect your existing CI/CD pipelines. Before upgrading the CLI, carefully read through [Upgrade to Astro CLI v1.0](upgrade-astro-cli.md) to learn more about these breaking changes and how they can affect your pipelines.

Reviewer: ```suggestion
Astro CLI v1.2.0 includes breaking changes that might effect your existing CI/CD pipelines. Before you upgrade the CLI, Astronomer recommends reading [Upgrade to Astro CLI v1.0](upgrade-astro-cli.md) to learn more about these breaking changes and how they can affect your pipelines.
```

Author: We should use the imperative here because this is a required task for all pre-1.0 users.

> Astro CLI v1.2.0 includes breaking changes that might effect your existing CI/CD pipelines. Before upgrading the CLI, carefully read through [Upgrade to Astro CLI v1.0](upgrade-astro-cli.md) to learn more about these breaking changes and how they can affect your pipelines.

Author: ```suggestion
Astro CLI v1.2.0 includes breaking changes that might effect your existing CI/CD pipelines. Before upgrading the CLI, read through [Upgrade to Astro CLI v1.0](upgrade-astro-cli.md) to learn more about these breaking changes and how they can affect your pipelines.
```

:::

> The Astro CLI is now a single CLI executable built for all Astronomer products. This new generation of the Astro CLI is optimized for a consistent local development experience and provides more value to Astronomer Software customers. For organizations moving from Astronomer Software to [Astro](https://docs.astronomer.io/astro), this change makes the transition easier.

Reviewer: How does this change "provide more value"? Be specific.

How does this change make the transition from Astronomer Software to Astro "easier"? Be specific. Provide a value message. For example:

_With the consolidated Astro CLI, your users no longer need to install and manage two different Astronomer command line tools. A single CLI simplifies the transition from Astronomer Software to Astro because your users will already be familiar with the Astro CLI. To learn more about Astro, see._
